### PR TITLE
restyle commit

### PR DESCRIPTION
**Pull request by imprecation**
restyle referenced:
https://github.com/feross/instant.io/issues/17#issuecomment-111589349

it uses font-awesome for 3 glyphs (github, upload icon and the rocket) and google fonts for fonts Poiret One and Lato.

### CODE REVIEW

**Comment by imprecation** [comment]
added footer and basic animation for hiding/showing torrent download form:
https://github.com/feross/instant.io/issues/17#issuecomment-111849044

animation used: https://gist.github.com/chrisbuttery/cf34533cbb30c95ff155

File: client/util.js [hunk 1]
```diff
@@ -36,3 +36,34 @@ exports.error = function error (err) {
   p.style.color = 'red'
   p.style.fontWeight = 'bold'
 }
+
+// https://gist.github.com/chrisbuttery/cf34533cbb30c95ff155
+exports.fadeOut = function fadeOut (el) {
+  el.style.opacity = 1;
```
Comment (imprecation) [inline client/util.js]: If I remove this semi-colon it causes an error

Comment (alxhotel) [inline client/util.js]: @imprecation Check out these rules from standard js: https://github.com/feross/standard/blob/master/RULES.md#semicolons

**Comment by feross** [comment]
I'm really busy right now -- will check this out when I get a chance. Thanks for sending!

**Comment by j8r** [comment]
@imprecation I've just installed your restyled version of Instant.io. Well done, nice design, i love it!

**Comment by feross** [comment]
Thanks for the PR @imprecation, but I'm going to use a design based on Google's Material design for this. I know it's a bummer that I'm not going to merge this, but I really appreciate the work you put into this. Thanks again!

**Pull request closed without merging**